js/src/dapps/registry/Names/names.js

```diff
@@ -72,13 +75,15 @@ const renderQueue = (queue) => {
   );
 };
 
-export default class Names extends Component {
+class Names extends Component {
 
   static propTypes = {
-    actions: PropTypes.object.isRequired,
     fee: PropTypes.object.isRequired,
     pending: PropTypes.bool.isRequired,
-    queue: PropTypes.array.isRequired
+    queue: PropTypes.array.isRequired,
+
+    reserve: PropTypes.func.isRequired,
```

Comment:
Naming here and elsewhere is confusing since it is not obvious what is actual functions and what is actual state variables. (queue here could be a call to queue as well)